plugin/core/views.py

```diff
@@ -689,3 +690,14 @@ def format_completion(
             details=st_details)
 
     return completion
+
+
+def code_lens_to_phantom(view: sublime.View, code_lens: CodeLens) -> sublime.Phantom:
+    region = range_to_region(Range.from_lsp(code_lens["range"]), view)
+    command = code_lens.get("command")
+    assert isinstance(command, dict)
+    return sublime.Phantom(region, command["title"], sublime.LAYOUT_BELOW)
```

Comment:
```
return sublime.Phantom(region, "<small>{}</small>".format(command["title"]), sublime.LAYOUT_BELOW)
```

Maybe wrap code lens with the small tag, so it is distinguishable from the code?

![image](https://user-images.githubusercontent.com/22029477/99980880-05811280-2da9-11eb-9c50-8d37b2841574.png)

Reply:
I would also argue to use the system font instead of the font of the editor.

It also looks like there are now multiple lenses stacked on top of the same starting point. But maybe they should be separated by comma's somehow (in a single, combined, phantom)